devel/mps_load_test.sh

```diff
@@ -208,7 +202,8 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
-make -j8 CONF=$compile_mode apache_trace_stress_test_server \
```

Comment:
I'm a bit confused. There's this in `devel/Makefile`:

```
apache_trace_stress_test_server :
        $(MAKE) apache_trace_stress_test_prepare
        $(MAKE) apache_debug_install
        $(MAKE) apache_debug_start
``` 

That is always going to expect libmod_pagespeed.so in `out/Debug/`, right? But the `libmod_pagespeed.so` we build will be ending up in `out/Release/`. So if the script succeeds, we're probably testing the wrong lib, but on a fresh checkout/build the script will fail.